### PR TITLE
Knife bootstrap flag comparison test.

### PR DESCRIPTION
**Pull request by ksubrama**
https://github.com/chef/oc_post_mortems/blob/master/2015/03/knife-windows-chef-12.md#corrective-actions

### CODE REVIEW

**Comment by ksubrama** [comment]
https://github.com/chef/chef/pull/3469 should fix the test fail against master.

File: lib/chef/knife/bootstrap_windows_ssh.rb [hunk 1]
```diff
@@ -60,14 +59,20 @@ class BootstrapWindowsSsh < Bootstrap
...
+
       option :identity_file,
         :short => "-i IDENTITY_FILE",
         :long => "--identity-file IDENTITY_FILE",
         :description => "The SSH identity file used for authentication"
 
-      option :host_key_verification,
-        :long => "--[no-]host-key-verification",
-        :description => "Disable host key verification",
+      option :host_key_verify,
```
Comment (btm) [inline lib/chef/knife/bootstrap_windows_ssh.rb]: If we change this we probably need some kind of backwards-compat / deprecation message?

File: spec/unit/knife/bootstrap_options_spec.rb [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    let(:opt_map) { {
+      :msi_url => :bootstrap_url,
+      :encrypted_data_bag_secret => :secret,
+      :encrypted_data_bag_secret_file => :secret_file,
+      :winrm_user => :ssh_user,
+      :winrm_password => :ssh_password,
+      :winrm_port => :ssh_port,
+      :winrm_ssl_verify_mode => :host_key_verify,
+    }}
+    let(:ref_ignore) { [
```
Comment (btm) [inline spec/unit/knife/bootstrap_options_spec.rb]: we should separate this into two arrays, one of things that we know are different and it's okay, and one that is a punch list of stuff we need to fix. and put a comment on them explaining that.

**Comment by jdmundrawala** [comment]
LGTM :+1:

**Pull request merged**